doc/advanced-topics.md

```diff
@@ -166,8 +210,9 @@ events in the IoT Agent with the configured type name will be marked as events.
 
 ##### Timestamp Processing Plugin (timestampProcess)
 
-This plugin processes the entity attributes looking for a TimeInstant attribute. If one is found, the plugin add a
-TimeInstant attribute as metadata for every other attribute in the same request.
+This plugin processes the entity attributes looking for a `TimeInstant` attribute. If one is found, for NGSI-v1/NGSIv2,
+the plugin adds a `TimeInstant` attribute as metadata for every other attribute in the same request. With NGSI-LD, the
+Standard `observedAt` metadata attribute is used instead.
```

Comment:
```suggestion
Standard `observedAt` property-of-a-property is used instead.
```

As far as I understand, in NGSI-LD the concept of metadata doesn't exist natively and the right term is property-of-a-preoperty (although I may be wrong).

Reply:
Well, technically a _property-of-a-property_ is the NGSI-LD representation of metadata, but I like the suggested amendments, since it serves to emphasise the distinction. I've accepted all of the docs changes suggestions.

Reply:
Fixed: 7991863